newsroom/wire/formatters/picture.py

```diff
@@ -27,7 +27,8 @@ def format_item(self, item, item_type="items"):
         if not picture:
             raise ValueError("Undefined picture")
 
-        picture_details = picture.get("renditions", {}).get("baseImage", {})
+        renditions = picture.get("renditions", {})
+        picture_details = renditions.get("original") or renditions.get("baseImage", {})
```

Comment:
we want to have it configurable, so can you pls add some `DOWNLOAD_RENDITION` config and use it here?

Reply:
okay